core/src/main/java/hudson/util/XStream2.java

```diff
@@ -131,7 +131,6 @@ protected Converter createDefaultConverter() {
      * @param clazz a class which we expect to hold a non-{@code transient} field
      * @param field a field name in that class
      */
-    @Restricted(NoExternalUse.class) // TODO could be opened up later
```

Comment:
Javadoc needs to be updated in order to indicate a version which starts offering this API publicly. `@since TODO` is a recommended way, the core maintainers will update the version during the merge after that.

Reply:
`@since TODO` added in the javadoc